network/vpc/troubleshooting/resource-attached-no-ip.mdx

```diff
@@ -0,0 +1,41 @@
+---
+meta:
+  title: My resource is attached to a Private Network but has no IP address on the network
+  description: This page helps you troubleshoot problems when you have a resource attached to a Private Network, but it has no reserved or configured IP private IP address on the network
+content:
+  h1: My resource is attached to a Private Network but has no IP address on the network
+  paragraph: This page helps you troubleshoot problems when you have a resource attached to a Private Network, wbut it has no reserved or configured IP private IP address on the network
+tags: resource instance private-network vpc ipv4 ipv6 ip-address IPAM reserved configured old nics 
+dates:
+  validation: 2023-07-05
+  posted: 2023-07-05
+categories:
+  - network
+---
+
+When you attach a resource to a Private Network, the Private Network's built in DHCP defines an IPv4 and IPv6 address for the resource on that network. This is managed via Scaleway's [internal IPAM](/network/vpc/concepts/#ipam). 
+
+- [Find out how to attach a resource to a Private Network](/network/vpc/concepts/#ipam/how-to/attach-resources-to-pn/)
+- [Find out how to view your resource's IP address on the Private Network](/network/vpc/how-to/attach-resources-to-pn/#how-to-view-the-resource's-ip-address)
+
+However, in some cases an IP address may not be set for your resource on the Private Network. Read on below to find out more about this problem, and how to solve it
+
+## The IP address is displayed, but is greyed out
+
+When you [view your resource's IP address on the Private Network](/network/vpc/how-to/attach-resources-to-pn/) you may notice it is greyed out, as in the example below:
+
+TODO SCREENSHOT
+
+This probably means you have attached an Instance or Elastic Metal server to an "old" Private Network on which you [have not activated DHCP](/network/vpc/reference-content/vpc-migration/#dhcp-activation-on-pre-existing-private-networks). In this case, a private IP address for the resource is reserved on Scaleway's [internal IPAM](/network/vpc/concepts/#ipam), but not configured on the resource itself. The IP address shown is therefore greyed out, and represents the reserved address only. 
+
+You will need to carry out manual configuration to set the IP address on the resource, or activate DHCP. 
+
+We strongly recommend that you activate DHCP on the Private Network to let us automatically handle IP address management. However, if you prefer to carry out manual configuration, we we suggest that you use the reserved IP address to avoid conflicts with our internal IPAM. Note that in this case, even if you configure the reserved IP address on the resource, it will remain greyed-out in the screen above.
```

Comment:
```suggestion
We strongly recommend that you activate DHCP on the Private Network to let us automatically handle IP address management. However, if you prefer to carry out manual configuration, we suggest that you use the reserved IP address to avoid conflicts with our internal IPAM. Note that in this case, even if you configure the reserved IP address on the resource, it will remain greyed-out in the screen above.
```